src/tests/Microsoft.PowerFx.Connectors.Tests/LivePublicSwaggerTests.cs

```diff
@@ -46,7 +42,8 @@ public async Task RealTest()
 
             Assert.True(ok);
 
-            FormulaValue result = engine.Eval(expr);
+            var runtimeConfig = new RuntimeConfig().AddService<IRuntimeConnectorContext>(new TestConnectorRuntimeContext("Math", client));
```

Comment:
>.AddService<IRuntimeConnectorContext>( [](http://example.com/codeflow?start=51&length=38)

nit - an extension method can help here and let you just write:


new RuntimeConfig().AddService(new TestConnectorRuntimeContext("Math", client));

That ensures they're getting the right service.

Reply:
I'd prefer AddConnectorContext(...) which is more explicit and less confusing than an overload

Reply:
I've added AddRuntimeContext on RuntimeConfig and BasicServiceProvider